installers/rpm-code/codes/mgis.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+mgis_main() {
+    codes_dependencies common boost fenics
+    codes_download thelfer/MFrontGenericInterfaceSupport
+    local f
+    for f in $(find . -name 'CMakeLists.txt'); do
```

Comment:
@robnagler thanks for your comment regarding us having support for c++17. This is the simplest hack I could find to get it working. `set(CMAKE_CXX_STANDARD 17)\nset(CXX_STANDARD_REQUIRED ON)` are in the top-level CMakeList.txt. But for an uknown (to me) reason that setting doesn't propagate down to the children CMakeLists.txt. This forces that to happen. I'm curious if there is a better way to do this?

Reply:
I do not know CMake. If you made it work, more power to you. :)